packages/shared/components/popups/LedgerLegacyTransaction.svelte

```diff
@@ -33,22 +33,26 @@
 </div>
 
 <div class="transaction flex flex-col space-y-4 scrollable-y">
-    <div class="rounded-lg bg-gray-50 dark:bg-gray-800 p-5 mb-4 flex flex-col space-y-1 text-center">
-        <Text type="h5" highlighted classes="mb-2">{outputString}</Text>
-        <Text type="pre">{formatUnitBestMatch(transfer.value)}</Text>
-        <Text type="pre">{formatAddressForLedger(transfer.address, true)}</Text>
-        <Text type="pre">
-            {#await asyncGetAddressChecksum(transfer.address)}...{:then checksum}{checksumString(checksum)}{/await}
-        </Text>
-    </div>
-    {#each inputs as { address, balance, index }}
-        <div class="rounded-lg bg-gray-50 dark:bg-gray-800 p-5 mb-4 flex flex-col space-y-1 text-center">
-            <Text type="h5" highlighted classes="mb-2">{inputString(index)}</Text>
-            <Text type="pre">{formatUnitBestMatch(balance)}</Text>
-            <Text type="pre">{formatAddressForLedger(address)}</Text>
+    <div class="mb-4">
```

Comment:
@maxwellmattryan just out of curiosity, why adding extra divs with `mb-4` if we already have all these elements wrapped in `flex space-y-4` which spaces them 16px?

Reply:
I was playing around with trying to get the divs all spaced evenly there. Before, the input ones were just a little closer together.